### PR TITLE
Adding note about MFA and CA policies to rooms-configure-accounts.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8213.

@dstrome, could you kindly confirm? Thanks.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/rooms/rooms-configure-accounts.md [hunk 1]
```diff
@@ -58,6 +58,10 @@ In hybrid environments the account used for Microsoft Teams Rooms must have pass
 You can think of a resource account as the resource account that people recognize as a conference room's or shared space's account. When you want to schedule a meeting using that space, you invite the account to that meeting.
   
 If you already have a resource mailbox account set up for the space where you're installing Microsoft Teams Rooms, you can change that account into a Teams Rooms resource account. Once that's done, all you need to do is sign in to Microsoft Teams Rooms with that account.
+
+> [!NOTE] 
+> As the Teams Rooms logs into Exchange in the background without user interaction, it cannot respond to any interactive prompts, such as MFA. The device account you create must be excluded from any such authentication requirements.
+> Teams Rooms are not compatible with Azure AD Conditional Access policies requiring device information (e.g. compliance). The device account that you create must be excluded from those CA policies.
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-configure-accounts.md:64]:
```suggestion
> As Teams Rooms logs into Exchange in the background without user interaction, it cannot respond to any interactive prompts, such as MFA. The device account you create must be excluded from any such authentication requirements.
>
> Teams Rooms are not compatible with Azure AD Conditional Access (CA) policies requiring device information (for example, compliance). The device account that you create must be excluded from those CA policies.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by dariomws** [comment]
Removed CA sentence:
https://www.microsoft.com/microsoft-365/roadmap?featureid=89068

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**